Update French localization

surfdude29 opened the pull request:
Following on from #4823, this PR adds translations for new strings, removes obsolete strings and makes a few tweaks.

I'd be grateful if you could take a look @Signez 🙏

Signez reviewed (commented):
Great translation work, again! A little bit more to say this time though. It can be summarised as: 

- `auteur·ice` should be used instead of `aut·eur·ice` ;
- `envoyer` should be used instead of `télécharger` (when it is used as a translation of `upload`); 
- some other more specific stuff.

Thanks again!

surfdude29 commented on src/locale/locales/fr/messages.po:
```diff
@@ -4923,23 +4757,23 @@ msgstr "Ratios"
 #: src/view/com/auth/SplashScreen.web.tsx:157
 msgid "Read the Bluesky blog"
-msgstr ""
+msgstr "Lisez le blog de Bluesky"
```
Annoyingly, my instinct was that it should be `Lire` but then I decided to change it 🙃

surfdude29 commented on src/locale/locales/fr/messages.po:
```diff
@@ -6423,7 +6189,7 @@ msgstr "Ce kit de démarrage n’a pas pu être trouvé."
 #: src/view/com/post-thread/PostQuotes.tsx:127
 msgid "That's all, folks!"
-msgstr ""
+msgstr "C’est tout, les ami·e·s !"
```
Simpler and shorter is good 👌

surfdude29 commented on src/locale/locales/fr/messages.po:
```diff
@@ -6487,7 +6253,7 @@ msgstr "Notre politique de confidentialité a été déplacée vers <0/>"
 #: src/state/queries/video/video.ts:222
 msgid "The selected video is larger than 100MB."
-msgstr ""
+msgstr "La vidéo sélectionnée a une taille supérieure à 100 Mo."
```
Good catch, I forgot that you need one between a number and the unit (like on Wikipedia).

surfdude29 commented on src/locale/locales/fr/messages.po:
```diff
@@ -6795,19 +6553,15 @@ msgstr "Ce compte ne suit personne."
 #: src/view/com/util/forms/PostDropdownBtn.tsx:689
 msgid "This will remove your post from this quote post for all users, and replace it with a placeholder."
-msgstr ""
+msgstr "Cela supprimera votre post de cette citation pour tous les comptes, et le remplacera par un espace réservé."
```
SGTM. Also, I noticed that the person doing the Italian translation didn't translate `placeholder` either, maybe it's just a difficult word to translate. Although tbf, I think the string in English would convey the meaning more clearly if it ended with `placeholder message.`

surfdude29 commented on src/locale/locales/fr/messages.po:
```diff
@@ -6840,16 +6590,12 @@ msgstr "Pour signaler une conversation, veuillez signaler un de ses messages via
 #: src/view/com/composer/videos/SelectVideoBtn.tsx:106
 msgid "To upload videos to Bluesky, you must first verify your email."
-msgstr ""
+msgstr "Pour télécharger des vidéos sur Bluesky, vous devez d’abord vérifier votre e-mail."
```
This one is fascinating, because I thought `télécharger` meant download, although I should have looked to see if `upload` has already been translated in another string (it probably has been, sorry).

But DeepL was **very** insistent that that was how `upload` (and related words) should be translated; I don't remember it ever showing me `envoyer` as an option.

Anyway, I promise to never, ever use `téléverser`, and `envoyer` does seem much clearer.

surfdude29 reviewed (commented):
Thanks so much again for all your corrections (and explanations) @Signez! :)

Sorry there were so many this time!

surfdude29 commented:
Just to double-check something: is `Rétablir le son` here the best translation for `Unmute`, or would `Activer le son` (like in L6108) work better? And similar question for the `Unmuted` string?

https://github.com/bluesky-social/social-app/blob/90682d43860388e63a458443425a4a43c48e5da9/src/locale/locales/fr/messages.po#L6106-L6108

https://github.com/bluesky-social/social-app/blob/90682d43860388e63a458443425a4a43c48e5da9/src/locale/locales/fr/messages.po#L6750-L6756

Also, I realise that I had already translated `Muted` – in the context of the sound in a video – as `Masqué` already, which seems like maybe not the clearest translation... should that be changed to something like `Son désactivé`?

https://github.com/bluesky-social/social-app/blob/90682d43860388e63a458443425a4a43c48e5da9/src/locale/locales/fr/messages.po#L3644-L3646

Signez commented:
> Just to double-check something: is `Rétablir le son` here the best translation for `Unmute`, or would `Activer le son` (like in L6108) work better? And similar question for the `Unmuted` string?

Both are fine actually. `Rétablir` is more like `get it back on`, which is great for `unmute`, so it makes sense to use it like you did.

It was not my choice in the `toggle` case but honestly it could: `désactiver ou rétablir le son` is better than `activer ou désactiver le son` (but please note that I changed the order of the two, like you would in English: `enable or disable the sound` vs. `unmute or get the sound back on` somehow?).


> Also, I realise that I had already translated `Muted` – in the context of the sound in a video – as `Masqué` already, which seems like maybe not the clearest translation... should that be changed to something like `Son désactivé`?

Good catch: sound can't be `masqué`, as it is specifically used for _hiding visually_. We don't really have an equivalent word in French (well, I don't have one readily) so `Son désactivé` is better, or `Son mis en sourdine` — but I am not a fan of the `sourdine` usage for anything else than sound so be careful. (if you remember, I made the active choice of not using `Sourdine` for muted accounts as it doesn't have a corresponding verb — you can't `sourdiner` something, you have to use `mettre en sourdine` which is clunky.)

Again, please don't ever be sorry for the amount of comments I had to make: it's always a pleasure for me to write them and it's fun to have to make explicit comments on my mother tongue 😅

(And `télécharger` DO mean `download`! It's the main usage of that word. It's the fact that it is sometimes overloaded with the `upload` meaning that is ambiguous — to be more precise, sometimes UI translators use the fact that technically an upload is a download looked from the server point of view to write `téléchager` where honestly we should use a simple `envoyer`, or `téléver…` NO WE SHOULD NOT USE THAT WEIRD WORD WE WON'T)

surfdude29 commented:
> > Just to double-check something: is `Rétablir le son` here the best translation for `Unmute`, or would `Activer le son` (like in L6108) work better? And similar question for the `Unmuted` string?
> 
> Both are fine actually. `Rétablir` is more like `get it back on`, which is great for `unmute`, so it makes sense to use it like you did.
> 
> It was not my choice in the `toggle` case but honestly it could: `désactiver ou rétablir le son` is better than `activer ou désactiver le son` (but please note that I changed the order of the two, like you would in English: `enable or disable the sound` vs. `unmute or get the sound back on` somehow?).
> 
> > Also, I realise that I had already translated `Muted` – in the context of the sound in a video – as `Masqué` already, which seems like maybe not the clearest translation... should that be changed to something like `Son désactivé`?
> 
> Good catch: sound can't be `masqué`, as it is specifically used for _hiding visually_. We don't really have an equivalent word in French (well, I don't have one readily) so `Son désactivé` is better, or `Son mis en sourdine` — but I am not a fan of the `sourdine` usage for anything else than sound so be careful. (if you remember, I made the active choice of not using `Sourdine` for muted accounts as it doesn't have a corresponding verb — you can't `sourdiner` something, you have to use `mettre en sourdine` which is clunky.)

Thanks, that's all very helpful! :)

I've changed the strings to reflect it like so:

https://github.com/bluesky-social/social-app/blob/2650d683376325b587bc46a33385cb4aa0c61c9f/src/locale/locales/fr/messages.po#L3644-L3646

https://github.com/bluesky-social/social-app/blob/2650d683376325b587bc46a33385cb4aa0c61c9f/src/locale/locales/fr/messages.po#L6106-L6108

https://github.com/bluesky-social/social-app/blob/2650d683376325b587bc46a33385cb4aa0c61c9f/src/locale/locales/fr/messages.po#L6750-L6756


Also, as a sidenote, I opened #5234 to ask that the labels for the `Mute` string be changed so that translators can distinguish between the uses in different contexts (muting a tag vs. muting a video):

https://github.com/bluesky-social/social-app/blob/2650d683376325b587bc46a33385cb4aa0c61c9f/src/locale/locales/fr/messages.po#L3571-L3574

> > Sorry there were so many this time!
> 
> Again, please don't ever be sorry for the amount of comments I had to make: it's always a pleasure for me to write them and it's fun to have to make explicit comments on my mother tongue 😅

I really appreciate all the comments, I was just a bit annoyed with myself for making so many silly mistakes 😅  Next time I think I'll give it a day or two and look at the translations again with fresh eyes before submitting the PR.

> > This one is fascinating, because I thought `télécharger` meant `download`, although I should have looked to see if `upload` has already been translated in another string (it probably has been, sorry).
> 
> Sorry if I wasn't clear, but `télécharger` DO mean `download`! It's the main usage of that word.

You were clear, I just didn't phrase my reply very clearly. What I was meaning to say was more like: "This one is fascinating, **because I knew that `télécharger` meant download**..." I never doubted that it meant download, I was just a bit surprised and confused that DeepL was insisting that it should also be used for upload.

> ...to be more precise, sometimes UI translators use the fact that technically an upload is a download looked from the server point of view to write `télécharger`...

I have to admit I never would have guessed at this logic 😆

Signez commented:
Great! Then I think that beside #5234, everything looks good to me. Thanks again for the amazing work 👍

surfdude29 commented:
Excellent 👌 Thanks again for your review comments! 🙏

surfdude29 commented:
I've updated the strings after #5234 and #5240 were merged.

Signez commented:
Perfect, LGTM then 👍

surfdude29 commented:
Merge and update string to reflect 50MB max file size following #5245.

celinelouvet commented on src/locale/locales/fr/messages.po:
```diff
@@ -7765,7 +7487,7 @@ msgstr "Vous avez atteint la fin"
 #: src/state/queries/video/video-upload.shared.ts:67
 msgid "You have temporarily reached the limit for video uploads. Please try again later."
-msgstr ""
+msgstr "Vous avez temporairement atteint la limite de téléchargement des vidéos. Veuillez réessayer plus tard."
```
It's like `many` and `much`. You can count the videos, but when you have uploaded `too much` of them, you should use `de vidéos`.

I don't know if that was very clear 😅
Yes, french is weird and too complex even for us...

surfdude29 replied:
Thanks @celinelouvet, that does make sense.

English can be pretty weird and complex for us native speakers too! 😅

surfdude29 commented:
I've added a translation for the new string that was added in #5273.

Signez commented:
Is there anything I can do to help you or the Bluesky team merging this one? I have to admit I would really like my French-speaking friend to enjoy the great work ~~you~~ we did in this PR, but I don't who (and if) I should ping to merge it.

surfdude29 commented:
Paul usually merges the localization update PRs right before he cuts a release. I don't know when they're planning the next release, but I imagine it won't be all that long to wait as I'm guessing there are quite a few bugfixes and improvements to video (along with other changes) that they want to get into users' hands.

You could ping him to ask him to merge it now, but even if it does get merged before a release your friend would only be able to use the updated translations on the web on the dev server ([main.bsky.dev](https://main.bsky.dev/)) and not on the iOS/Android app.

Signez commented:
Oh, right. There is no hurry to be honest (and I forgot an plural mark at "friends" in my previous comment, it was more of a general thought), it can clearly wait for a week or two. And I am sure Paul have other stuff to worry about these days 😅

surfdude29 commented:
Yeah hopefully it won't be too long to wait 🤞🦋

surfdude29 commented:
I added the new usage of `Follow Back` and I translated the new string `Similar accounts` as `Comptes similaires`.

Signez commented on src/locale/locales/fr/messages.po:
```diff
@@ -844,6 +844,10 @@ msgstr "Bluesky est un réseau ouvert où vous pouvez choisir votre hébergeur.
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:206
+msgid "Bluesky now has over 10 million users, and I was #{0}!"
+msgstr "Bluesky comprend aujourd’hui plus de 10 millions de comptes, et j’étais n° {0} !"
```
Reversing the structure helps us remove the `comptes` and makes it punchier.

```suggestion
msgstr "Nous sommes désormais plus de 10 millions sur Bluesky, et j’étais n°{0} !"
```

Signez commented on src/locale/locales/fr/messages.po:
```diff
@@ -865,6 +869,10 @@ msgstr "Flouter les images et les filtrer des fils d’actu"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:614
+msgid "Brag a little!"
+msgstr "Vantez un peu !"
```
Yikes, that would work in the 1930s I think but not today. The most equivalent would be `Crânez un peu !` but it's too familiar, or `Fanfaronnez-donc !` but it would be WAY too formal (it would work if you said that in a French embassy, maybe?).

Let's workaround by saying basically the same thing but using another structure, yet again.

```suggestion
msgstr "Soyez-en fier !"
```

surfdude29 replied:
SGTM

Signez commented on src/locale/locales/fr/messages.po:
```diff
@@ -1004,6 +1012,10 @@ msgstr "Sous-titres (.vtt)"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:369
+msgid "Celebrating {0} users"
+msgstr "Célébration de {0} de comptes"
```
Translating a `-ing` form as a noun is good most of the time, but in _that_ case, it would make it lose its force drastically. Let's be more "active".

```suggestion
msgstr "Célébrons les {0} comptes"
```

Signez commented on src/locale/locales/fr/messages.po:
```diff
@@ -3111,6 +3127,10 @@ msgstr "Rejoignez Bluesky"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:493
+msgid "Joined {0}"
+msgstr "Inscrit le {0}"
```
Even though we could say it's _technically_ the `compte` that was `inscrit`, it wouldn't work here. Also, adjective are very rarely _épicène_, so I am afraid that the _point médian_ is the only way I can find (especially in my bed at 1am). 

```suggestion
msgstr "Inscrit·e le {0}"
```

surfdude29 replied:
Oops, completely forgot about _la langue épicène_ in this context 😬

Signez commented on src/locale/locales/fr/messages.po:
```diff
@@ -5677,6 +5701,14 @@ msgstr "Partager quand même"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:621
+msgid "Share image externally"
+msgstr "Partager l’image avec d’autres personnes"
```
Well, yeah, but, _ailleurs_ is shorter and works well in that specific context imho.

```suggestion
msgstr "Partager l’image ailleurs"
```

Signez commented on src/locale/locales/fr/messages.po:
```diff
@@ -6610,6 +6650,10 @@ msgstr "Pour envoyer des vidéos sur Bluesky, vous devez d’abord vérifier vot
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:598
+msgid "Together, we're rebuilding the social internet. We're glad you're here."
+msgstr "Ensemble, nous reconstruisons l’internet social. Nous sommes heureux que vous soyez là."
```
It's weird, but we say more often `web social` than `internet social` in French, so I think it's better to use that term. Also, `glad` is more `ravis` than `heureux`, but honestly `heureux` would be fine; to make sure that `here` is understood as `among us` (because `là` could be understood as `there`, beside `here`), let's translate `among us` directly.

```suggestion
msgstr "Ensemble, nous reconstruisons le web social. Nous sommes ravis que vous soyez parmi nous."
```

Signez reviewed (commented):
🎉

surfdude29 reviewed (commented):
All suggestions applied, thanks again! :)

surfdude29 commented:
I made a few updates following #5340 and #5343, hopefully pretty straightforward.

Signez commented:
LGTM 👍

haileyok reviewed (approved):
Thanks everyone!
